tests/functions_test.go

```diff
@@ -178,17 +179,17 @@ func TestCheckPrivilegesGrantedAccess(t *testing.T) {
 
 	cred := jwt.Credential{KeyId: "", APIKey: "fake_api_key", AccessKey: "non_exprired_token", APIEndpoint: "http://www.test.com"}
 
-	grantedAccessJson := "{\"project_access\": " +
-		"{\"test_project\": {" +
-		"\"0\": \"read\"," +
-		"\"1\": \"create\"," +
-		"\"2\": \"read-storage\"," +
-		"\"3\": \"update\"," +
-		"\"4\": \"delete\"}" +
-		"}}"
+	grantedAccessJSON := "{\"project_access\": " +
```

Comment:
```suggestion
	grantedAccessJSON := `{
		"project_access": {
			"test_project": ["read","create","read-storage","update","delete"]
		}
	}`
```

Not really related to this PR, but template strings might make this JSON a little cleaner 🧹